benchmark/nixlbench/src/utils/utils.cpp

```diff
@@ -55,6 +55,9 @@ DEFINE_uint64(max_block_size, 64 * (1 << 20), "Max size of block \
 DEFINE_uint64(start_batch_size, 1, "Starting size of batch (Default: 1)");
 DEFINE_uint64(max_batch_size, 1, "Max size of batch (starts from 1)");
 DEFINE_int32(num_iter, 1000, "Max iterations");
+DEFINE_int32 (large_block_iter_factor,
+              1,
+              "factor to reduce test iteration when testing large block size(>1MB)");
```

Comment:
@Pyngon, as we talked about offline, setting the default value for large_block_iter_factor to 1 will enable 1000 iterations for the large block sizes. It consumes too much time for comm benchmarks, which ideally saturates the b/w much before for large sizes.

Let's check with @w1ldptr and @brminich.